l10n_it_fiscalcode/__openerp__.py

```diff
@@ -42,16 +30,10 @@
         'python': ['codicefiscale'],
     },
     'data': [
-        'view/fiscalcode_view.xml',
-        'wizard/compute_fc_view.xml',
-        'data/res.city.it.code.csv',
-        "security/ir.model.access.csv"
+        'view/fiscalcode_view.xml'
         ],
     'qweb': [],
+    'images': [],
```

Comment:
oca bot throw a warning because images is required tag .. remove qweb tag but let images ...